invenio_pidstore/__init__.py

```diff
@@ -328,7 +328,7 @@
 PIDStore will automatically register minters and fetchers defined by the entry
 point groups ``invenio_pidstore.minters`` and . ``invenio_pidstore.fetchers``.
 
-Example:
+Example code:
```

Comment:
The word "Example" was treated as a title  of a section and it was causing the formatting issues so I needed to change it.